aten/src/ATen/native/quantized/cpu/qembeddingbag.cpp

```diff
@@ -957,6 +957,29 @@ Tensor embedding_bag_2bit_rowwise_offsets(
   return output;
 }
 
+Tensor embedding_bag_byte_rowwise_offsets_meta(
+    const Tensor& weight,
+    const Tensor& /* indices */,
+    const c10::optional<Tensor>& offsets_in,
+    const bool /* scale_grad_by_freq */,
+    const int64_t /* mode */,
```

Comment:
Can you add asserts to make sure the assumptions to made on arguments like `mode` are explicitly enforced?

Reply:
@gmagogsfm Updated:
* Validated that `mode` and `scale_grad_by_freq` [aren't used at all in the actual op](https://github.com/pytorch/pytorch/blob/main/aten/src/ATen/native/quantized/cpu/qembeddingbag.cpp#L889-L890)
* Brought over some checks for `indices` based on ndims/dtypes, and also offsets dtype
* Validated `per_sample_weights_`, `pruned_weights`, and `compressed_indices_mapping` make no difference on the output -- entirely internal to the kernel.